cpp/open3d/t/pipelines/odometry/RGBDOdometry.h

```diff
@@ -263,19 +275,19 @@ OdometryResult ComputeOdometryResultIntensity(
 /// image obtained by RGBToGray before calling this function.
 /// \param target_intensity (rows, cols, channels=1) Float32 target intensity
 /// image obtained by RGBToGray before calling this function.
-/// \param source_depth_dx (rows, cols, channels=1) Float32 source depth
-/// gradient image at x-axis obtained by FilterSobel before calling this
+/// \param target_depth_dx (rows, cols, channels=1) Float32 target depth
```

Comment:
@theNded The `source_*` and `target_*` might have been interchanged in this file. I found them to be opposite in `.cpp`, may you please verify this in the new changes, if they are fixed?

Reply:
Done.